### PR TITLE
New package: dkimproxy-1.4.1.

### PR DESCRIPTION
**Pull request by jnbr**
#6359

### CODE REVIEW

File: srcpkgs/dkimproxy/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'dkimproxy'
+pkgname=dkimproxy
+version=1.4.1
+revision=1
+build_style=gnu-configure
+noarch=yes
```
Comment (the-maldridge) [inline srcpkgs/dkimproxy/template]: This is almost certainly wrong for something that uses gnu-configure build style.

Comment (jnbr) [reply srcpkgs/dkimproxy/template]: The configure script doesn't do anything architecture specific. It is mainly used for installing files into the correct location. The only changes it does to the scripts, is setting the include path for perl.
For example:
```
--- scripts/dkim_responder.pl   2010-01-22 15:53:24.000000000 +0100
+++ /usr/bin/dkim_responder.pl  2018-03-10 03:58:23.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -I../lib
+#!/usr/bin/perl -I/usr/lib
 #
```
The same happens to `/usr/bin/dkimproxy.in` and `/usr/bin/dkimproxy.out`.
Do you still want the `noarch=yes` to be removed?

Comment (cr6git) [reply srcpkgs/dkimproxy/template]: @the-maldridge the Debian [package](https://packages.debian.org/sid/dkimproxy) and the Arch [package](https://www.archlinux.org/packages/community/any/dkimproxy/) are `noarch`.

Comment (the-maldridge) [reply srcpkgs/dkimproxy/template]: @jnbr if this is actually perl and not C then perhaps the perl-module style is more apt, if not put a comment above that that this is perl and not C.

Comment (jnbr) [reply srcpkgs/dkimproxy/template]: @the-maldridge yes, it is pure perl. I added a comment.

**Comment by jnbr** [comment]
@the-maldridge is the comment about noarch clear enough or do you want it to be changed?

**Pull request closed without merging**